Src/Shared/Core/Active.cpp

```diff
@@ -88,14 +88,14 @@ void CActive::CheckForAndThrowException() const
 {
 	if(_isDone)
 	{
-		if(_activeException)
+		if (std::exception_ptr exception = _activeException)
```

Comment:
Wow, I was not aware variable declarations inside if-conditions where already a thing.